tests/emit.spec.ts

```diff
@@ -316,7 +316,7 @@ describe('emitted', () => {
     })
 
     const wrapper = mount(Foo)
-    await wrapper.trigger('click')
+    await wrapper.get('div').trigger('click')
```

Comment:
This looks wrong, we should be able to keep the previous behavior.

Reply:
Agreed